### PR TITLE
all changes have been made

### PR DESCRIPTION
**Pull request by Thirulogachandar**
all changes are made. all the pages

### CODE REVIEW

**Review by jayasuryajeyakodi — APPROVED** [review]

File: temp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<div data-element-id="elm_38IwJLKLiSAaHrAAQ0QzsQ" data-element-type="column"
```
Comment (boopeshmahendran227) [inline temp:1]: We can remove this file, I guess.

File: ourwork.html [hunk 2]
```diff
@@ -1608,9 +1610,17 @@
...
         </div>
     </div>
-    <script type="text/javascript">
+    <div style="position:fixed; bottom:10px;right:0;
+    border-top: 50px solid transparent;
+    border-bottom: 50px solid transparent; border-right:50px solid rgb(223, 103, 23);">
+
+    </div>
+    <div style="background-color: rgb(223, 103, 23); position:fixed; bottom:30px;right:0; height: 60px; width: 60px;border-bottom-left-radius: 10px;border-top-left-radius:10px;  padding-top:10px;padding-left:10px; " >
+        <a href="http://api.whatsapp.com/send?phone=919381069919&amp;text=HELLO%20HungerfreeIndiaFoundation"><i style="font-size: 40px;"  class="material-icons">chat</i></a>
```
Comment (boopeshmahendran227) [inline ourwork.html:1619]: We should change the phone number to correct phone number and also in other pages.

File: donate.html [hunk 3]
```diff
@@ -238,45 +214,64 @@
...
+                                                        year.&nbsp;</span><span>Donating Monthly is a great way to help
+                                                        more children.</span></div>
+                                            </span>
+                                            <div>
+                                                <div style="text-align:left;"><span style="font-size:12px;"><br></span>
+                                                </div>
+                                                <div style="text-align:left;">
+                                                    <div><span style="font-size:16px;"><span
+                                                                style="color:inherit;">Join</span>&nbsp;our Team Everest
+                                                            HOPE monthly donor community and support a child
```
Comment (boopeshmahendran227) [inline donate.html:266]: We have to change `Team Everest HOPE` to `Hunger Free India Foundation` here and also in other places.

File: css/styles.css [hunk 4]
```diff
@@ -46,3 +46,18 @@
   height: 100%;
   background: rgba(0, 0, 0, 0.4);
 }
+
+.account-card {
+
```
Comment (boopeshmahendran227) [inline css/styles.css:51]: We can remove these extra new lines.

**Review by boopeshmahendran227 — COMMENTED** [review]
Looks good 🙂

**Pull request merged**